ruby23/plan.sh

```diff
@@ -13,3 +13,6 @@ pkg_source=https://cache.ruby-lang.org/pub/ruby/ruby-${pkg_version}.tar.gz
 pkg_upstream_url=https://www.ruby-lang.org/en/
 pkg_shasum=b5016d61440e939045d4e22979e04708ed6c8e1c52e7edb2553cf40b73c59abf
 pkg_dirname="ruby-$pkg_version"
+
+# Hint for rebuild scripts. Not a formal part of plan-build.
+_pkg_deprecated="true"
```

Comment:
We could add a CI check to not fail for a missing .bldr.toml entry if this `_pkg_deprecated="true"` exists.
I can add that - but is that logic sound?

Reply:
I think before we do that, we should make `_pkg_deprecated` the correct way through a design document as the current document doesn't cover it. I've had it on my back-burner for a while now to address this, but it keeps getting pushed back because deprecations are rare.   I'll try and get that done in the next week.